e2e/__tests__/typescriptConfigFile.test.ts

```diff
@@ -20,7 +24,7 @@ test('works with single typescript config that imports something', () => {
     '__tests__/mytest.alpha.js': "test('alpha', () => expect(1).toBe(1));",
     '__tests__/mytest.common.js': "test('common', () => expect(1).toBe(1));",
     'alpha.config.ts': `
-    import commonRegex from './common';
+    import commonRegex from './common${importFileExtension}';
```

Comment:
transpiled resolution is fine with `'./common'`, type-stripped node execution wants `'./common.ts'`.